expected-consensus.md

```diff
@@ -333,9 +333,17 @@ TODO: ensure 'power' is properly and clearly defined
 
 It is possible for forks to emerge naturally in Expected Consensus. EC relies on weighted chains in order to quickly converge on 'one true chain', with every block adding to the chain's weight. This means the heaviest chain should reflect the most amount of work performed, or in Filecoin's case, the most storage provided.
 
-The weight at each block is equal to its `ParentWeight`, plus that block's delta weight. Delta
-weight is a constant `V`, plus `X` - a function of the total power in the network as reported in the Power Table.  The exact value for `V` and the magnitude of the power ratio value are
-still to be determined, but for now `V = 10` and `X = log(TotalPower)`.
+The weight at each block is equal to its `ParentWeight` plus that block's delta weight. Delta
+weight is a term P times the sum of a constant `V` times the number of blocks included in that TipSet in the given round (can only be calculated in the next round), and `X` - a function of the total power in the network as reported in the Power Table. P will be equal to some term P_i^P_n and seeks to punish selfish mining. So we have:
+
+`Weight = ParentWeight + P(V*NumBlocksMinedInThisRound + X)`
+
+ The exact value for these parameters remain to be determined, but for now we set:
+
+- `V = 2 * number of blocks in the round`
+- `X = log(TotalPower in Power Table l blocks back)`
+- `P_i = .87`
+- `P_n = the number of tickets in this block's Ticket array` if that number is  >= 3, 0 otherwise.
```

Comment:
Something we need to keep in mind: keeping infinite precision in our weight is not feasible.  Right now we only track a delta weight >= 1/1000 [see here](https://github.com/filecoin-project/go-filecoin/blob/master/types/fixed_point.go#L5)

It turns out that we don't hit this fraction until after 50 null blocks ( (0.87)^50 < 1/1000) so this is not a big concern.  The implementation must round these events up to a delta of 1/1000 to have guaranteed liveness throughout time however.

Reply:
I did not know that. I will work to adapt the functions accordingly. Thank you!